tslint.json

```diff
@@ -1,6 +1,11 @@
 {
```

Comment:
The file above `svic.jpg` is not in fixtures folder as well as `placeholder-img.jpg`.